### PR TITLE
PrestaShop 1.7 compatibility

### PR DESCRIPTION
**Pull request by firstred**

### CODE REVIEW

**Comment by firstred** [comment]
Hi @Quetzacoalt91 :wave:

File: followup.php [hunk 1]
```diff
@@ -435,352 +567,407 @@ public function renderStats()
...
+     * @return bool
+     * @throws PrestaShopDatabaseException
+     *
+     * @since 2.4.0
+     */
+    protected function updateConfiguration()
+    {
+        foreach ($this->getConfigFields() as $field) {
+            if (!Tools::isSubmit($field)) {
+                return false;
```
Comment (Quetzacoalt91) [inline followup.php]: Shouldn't we `continue;` instead?

**Comment by Quetzacoalt91** [comment]
@LouiseBonnard, can you please take some time to review the different strings used in this module?

**Comment by LouiseBonnard** [comment]
@Quetzacoalt91, I'll have a look at it!

**Comment by Quetzacoalt91** [comment]
I've been reminded that there is already a 1.7 equivalent module with another name: https://github.com/PrestaShop/ps_reminder. It seems we do not only make these module compliant with PS 1.7, but also plan to improve the features on these modules.
I'll add a list of equivalence in the devdocs to prevent further mistakes.

I'm sorry about that @firstred. If we already split the modules regarding their compatibility, this PR may not be accepted here. :neutral_face:

**Comment by firstred** [comment]
Interesting. And how does that work with upgrades from 1.6?

**Comment by Quetzacoalt91** [comment]
We have to handle them manually, as I did for blockcmsinfo >> ps_customtext. (https://github.com/PrestaShop/ps_customtext/pull/13)
That's why I'd like to bring back the 1.7 to the same technical name, this would make the upgrade drastically easier.

**Pull request closed without merging**